api/v1alpha1/operator_types.go

```diff
@@ -46,34 +46,43 @@ type OperatorSpec struct {
 
 const (
 	// TODO(user): add more Types, here and into init()
-	TypeReady = "Ready"
+	TypeReady    = "Ready"
+	TypeResolved = "Resolved"
 
 	ReasonInstallationSucceeded     = "InstallationSucceeded"
 	ReasonResolutionFailed          = "ResolutionFailed"
+	ReasonResolutionUnknown         = "ResolutionUnknown"
```

Comment:
Nit: these ought to be alphabetical, but others are not. :(

Reply:
fixed.